tests/mutations/test_mutations.py

```diff
@@ -95,3 +97,245 @@ def test_delete_with_filters(mutation, fruits):
     assert list(models.Fruit.objects.values("id", "name")) == [
         {"id": 3, "name": "banana"},
     ]
+
+
+@pytest.mark.skipif(
+    not settings.GEOS_IMPORTED,
+    reason="Test requires GEOS to be imported and properly configured",
+)
+@pytest.mark.django_db(transaction=True)
+def test_create_geo(mutation):
+    from tests.models import GeosFieldsModel
+
+    # Test for point
+    point = [0.0, 1.0]
+    result = mutation(
+        "{ geofield: createGeoField(data: { point: " + str(point) + " }) { id } }"
+    )
+    assert not result.errors
+    assert (
+        deep_tuple_to_list(
+            GeosFieldsModel.objects.get(id=result.data["geofield"]["id"]).point.tuple
+        )
+        == point
+    )
+
+    # Test for lineString
+    line_string = [[0.0, 0.0], [1.0, 1.0]]
+    result = mutation(
+        "{ geofield: createGeoField(data: { lineString: "
+        + str(line_string)
+        + " }) { id } }"
```

Comment:
You could use `.format()` here, but better yet, pass `line_string` as `variable_values`

Reply:
Done d6584a21713a259f4513278df8f0afb19b27131c

Reply:
I still think that tuples should be used and `deep_tuple_to_list` should not be used.

If you are having trouble with this, feel free to hit me on discord so that I can help you with that